nodejs/week3/calculatorRouter.js

```diff
@@ -0,0 +1,70 @@
+
+const express = require("express");
+const calculatorRouter = express.Router();
+
+// /calculator/addition?firstParam=1&secondParam=2 should respond with 3.
+
+calculatorRouter.get('/addition', (request,response) => {
+
+  console.log(request.query);
+  if (request.query.firstParam === {num1}, request.query.secondParam === {num2} ) { 
+
+  // New app using express module 
+  const num1 = parseInt(request.params.num1);
+  const num2 = parseInt(request.params.num2);
+  
+  var result =  num1 + num2;
+  
+  response.send(result); 
+};});
+
+// /calculator/subtration?firstParam=4&secondParam=2 should respond with 2.
+
+calculatorRouter.get('/subtration', (request,response) => {
+  console.log(request.query);
+  if (request.query.firstParam === {num1}, request.query.secondParam === {num2} ) { 
+
+  // New app using express module 
+
+  const num1 = parseInt(request.params.num1);
```

Comment:
When assigning a variable like this that is only used once, I personally prefer to not assign it and just use an inline variable. More can be read about inline variable [here](https://refactoring.com/catalog/inlineVariable.html).

To be more specific, instead of doing the:

    const num1 = parseInt(request.params.num1);

and then:

    var result =  num1 - num2;

you can do:

    var result = parseInt(request.params.num1) - parseInt(request.params.num2);

Also, make sure you are using `let` instead of `var`.